src/pages/Final.less

```diff
@@ -0,0 +1,15 @@
+.final-page {
+    h1,
+    p {
+        text-align: center;
+    }
+
+    p {
+        margin-bottom: 20px;
```

Comment:
Why not using Bootstrap defaults?

Reply:
it doesn't look good

Reply:
I understood we shouldn't take design considerations beyond the Bootstrap defaults. Otherwise, further design iterations will be way more difficult to achieve.